backend/assignments/1/src/main/java/shrishti/example/Query.java

```diff
@@ -0,0 +1,85 @@
+package shrishti.example;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import shrishti.example.Logger.logger;
+
+public class Query {
+    public static void retrieveCoinDetails(String coinName) {
+        Coin c = null;
```

Comment:
Add better names for declarations. 
They should denote the purpose of DS/function/variable etc